.gitignore

```diff
@@ -125,4 +125,5 @@ chromedriver*
 logs.txt
 nohup.out
 gunicorn*.log
-pytestdebug.log
\ No newline at end of file
+pytestdebug.log
+deploy-to-heroku.ps1
```

Comment:
Really minor, but this looks like a file that you might experiment with locally, but which other developers would never expect to see. Ignoring it to avoid checking it in makes sense, but if it's only relevant to a single developer you might choose to ignore it locally rather than for the entire repo; [see this SO answer for various ways we can do that](https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally) - I tend to use the exclude file.